specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/2017-03-15-preview/OperationalInsights.json

```diff
@@ -1104,6 +1104,45 @@
       ],
       "description": "The SKU (tier) of a workspace."
     },
+    "Features": {
+      "properties": {
+        "legacy": {
+          "type": "string",
+          "description": "The legacy status of the workspace.",
+          "enum": [
+            "None",
+            "GrandfatheredNonSuite",
+            "GrandfatheredSuite"
+          ],
+          "x-ms-enum": {
+            "name": "legacyNameEnum",
+            "modelAsString": true
+          }
+        },
+        "searchVersion": {
+          "type": "string",
+          "description": "The search version of the workspace.",
+          "enum": [
+            "OQL",
+            "KQL"
+          ],
+          "x-ms-enum": {
+            "name": "searchVersionNameEnum",
+            "modelAsString": true
+          }
+        },
+        "searchVersionUpgradeTime": {
```

Comment:
Is this a readonly property generated by the service? Or something the user needs to provide? It is not clear.